src/AVFoundation/AVPlayerItemVideoOutput.cs

```diff
@@ -13,24 +13,24 @@ public partial class AVPlayerItemVideoOutput {
 		enum InitMode {
 			PixelAttributes,
 			OutputSettings
-		} 
+		}
 
-		AVPlayerItemVideoOutput (NSDictionary data, AVPlayerItemVideoOutput.InitMode mode) : this (IntPtr.Zero)
+		AVPlayerItemVideoOutput (NSDictionary data, AVPlayerItemVideoOutput.InitMode mode) : this ()
```

Comment:
Shouldn't this call `: base (NSObjectFlag.Empty)` instead?

Otherwise we'll be doing this:

    [[[AVPlayerItemVideoOutput] alloc] init] initWith..]

i.e. calling `init*` twice.